apps/templates/pages/labs_view.html

```diff
@@ -58,9 +77,15 @@ <h1>View Labs</h1>
             {% for cat in lab.categories %}
               {% set _ = category_classes.append("category-" ~ cat.id) %}
             {% endfor %}
-            <div class="card card-dark filter {{category_classes|join(' ')}}">
+            <div class="card card-dark filter {{category_classes|join(' ')}} {% if lab.id in completed_labs %}lab-completed{% else %}lab-not-completed{% endif %}">
```

Comment:
following the idea from previous comment, here we could have a tag for "running" and the other you added above for "completed".